Makefile

```
@@ -0,0 +1,8 @@
+.PHONY: generate
+generate:
+	# Generate the types from the JSON schema
+	go-jsonschema --only-models -p migrations --tags json schema.json > pkg/migrations/types.go
```

We could run `go-jsonschema` via docker to minimize dependencies and ensure a consistent version:

```suggestion
	docker run -v $$PWD/schema.json:/mnt/schema.json omissis/go-jsonschema:0.14.1 --only-models -p migrations --tags json /mnt/schema.json > pkg/migrations/types.go
```